docs/getting-started.md

```diff
@@ -0,0 +1,179 @@
+- To persist and view results outside of Airflow, we recommend using a Cloud Data Context. Validation Results are automatically visible in the GX Cloud UI when using a Cloud Data Context, and the task result contains a link to the stored validation result. All 3 Operators support the Cloud Data Context.
+- If you want to manage Validation Results yourself, use a File Data Context. With this option, Validation Results can be viewed in [Data Docs](https://docs.greatexpectations.io/docs/core/configure_project_settings/configure_data_docs/). Only the `GXValidateCheckpointOperator` supports the File Data Context.
+
+## Prerequisites
+
+- [Python](https://www.python.org/) version 3.9 to 3.12
+- [Great Expectations](https://docs.greatexpectations.io/docs/core/set_up_a_gx_environment/install_gx) version 1.3.3+
+- [Apache Airflow®](https://airflow.apache.org/) version 2.1.0+ 
+
+## Assumed knowledge
+
+To get the most out of this getting started guide, make sure you have an understanding of:
+
+- The basics of Great Expectations. See [Try GX Core](https://docs.greatexpectations.io/docs/core/introduction/try_gx/).
+- Airflow fundamentals, such as writing DAGs and defining tasks. See [Get started with Apache Airflow](https://www.astronomer.io/docs/learn/get-started-with-airflow/).
+- Airflow Operators. See [Operators 101](https://www.astronomer.io/docs/learn/what-is-an-operator/).
+- Airflow connections. See [Managing your Connections in Apache Airflow](https://www.astronomer.io/docs/learn/connections/).
+
+## Install the provider and dependencies
+
+1. Install the provider.
+
+   ```bash
+   pip install airflow-provider-great-expectations 
+   ```
+2. (Optional) Install additional dependencies for the data sources you’ll use. For example, to install the optional Snowflake dependency, use the following command.
+
+   ```bash
+   pip install "airflow-provider-great-expectations[snowflake]"
+   ```
+   The following backends are supported as optional dependencies:
+      - `athena`
+      - `azure`
+      - `bigquery`
+      - `gcp`
+      - `mssql`
+      - `postgresql`
+      - `s3`
+      - `snowflake`
+      - `spark`
+
+## Configure an Operator
+
+After deciding [which Operator best fits your use case](#operator-use-cases), follow the Operator-specific instructions below to configure it.
+
+### Data Frame Operator
+
+1. Import the Operator.
+
+    ```python
+    from great_expectations_provider.operators.validate_dataframe import (
+        GXValidateDataFrameOperator,
+    )
+    ```
+
+2. Instantiate the Operator with required and optional parameters.
+
+    ```python
+    my_data_frame_validation = GXValidateDataFrameOperator(
+        task_id="my_data_frame_validation",
+        configure_dataframe=lambda: pd.read_csv(my_data_file),
+        expect=my_expectation_suite,
+    )
+    ```
+
+    - **`task_id`**: alphanumeric display name for the task in the Airflow UI.
```

Comment:
it may be worth mentioning that this name will also be used by the operator to name GX resources. It's not really relevant for ephemeral users, since they will never see the GX components backing the validation, but it _is_ relevant for cloud users.